pkg/apis/directpv.min.io/v1beta1/drive.go

```diff
@@ -221,6 +222,24 @@ func (drive DirectPVDrive) GetNodeID() types.NodeID {
 	return types.NodeID(drive.getLabel(types.NodeLabelKey))
 }
 
+// HasUniqueAllocID checks if the provided allocation id is set on the drive.
+func (drive *DirectPVDrive) HasUniqueAllocID(allocID string) bool {
+	if v, ok := drive.GetLabels()[types.UniqueAllocIDLabelKeyPrefix+allocID]; ok && v == strconv.FormatBool(true) {
+		return true
+	}
+	return false
```

Comment:
You could simplify this by `return drive.GetLabels()["directpv.min.io/volume-claim-id-555e99eb-e255-4407-83e3-fc443bf20f86"] != ""`
```
```

Reply:
The check is like `return drive.GetLabels()[types.VolumeClaimIDLabelKeyPrefix+claimID] == strconv.FormatBool(true)`